content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -172,6 +203,26 @@ ResourceClaims, and thus scheduling the next pod gets delayed.
 
 {{< /note >}}
 
+### With structured parameters
+
+When a driver uses structured parameters, the scheduler takes over the
+responsibility of allocating a ResourceClaim once a pod needs the
+ResourceClaim. It does that by retrieving the information about available
+resources from ResourceSlice objects, tracking which of those are in use based
+on already allocated ResourceClaims, and then selecting resources which match
+the requirements defined by ResourceClaimParameters and (if specified)
+ResourceClassParameters and which are available on a potential node.
```

Comment:
```suggestion
When a driver uses structured parameters, the scheduler takes over the
responsibility of allocating resources to a ResourceClaim whenever a pod needs them. It does so by retrieving the full list of available
resources from ResourceSlice objects, tracking which of those resources have already been allocated to existing ResourceClaims, and then selecting from those resources that remain.  The exact resources selected are subject to the constraints provided in any ResourceClaimParameters or
ResourceClassParameters associated with the claim.
```

Reply:
s/claim/ResourceClaim/, for consistency.